### ISSUE TITLE
A way to merge back the "altered" values into the .less?

### ISSUE THREAD

**Issue by matthew-dean** [css-george/less-plugin-css-george#3]
This is an interesting project!  I imagine the advantage of doing this is that the values can be changed much quicker to see the result (vs `modifyVars` in Less?). My main question though- after you change the values in CSS George, can you merge them back to the Less stylesheet? To do something like create a `variables.less` file?

**Comment by dpogue** [comment]
Yes, one unfinished task is to add some sort of export functionality to the CSS George editor, which could export a JSON file or Less file containing the modified values.

It's not actually a difficult thing to do, so I should probably stop procrastinating on it :sweat_smile:
